testsuite/forge-cli/src/main.rs

```diff
@@ -41,7 +42,7 @@ struct Args {
     ops_cmd: Option<OperatorCommand>,
 
     #[structopt(long, help = "Name of the EKS cluster")]
-    cluster_name: String,
+    cluster_name: Option<String>,
```

Comment:
Since the CLI is getting more complicated, can we maybe change the way we invoke tests?
Instead of `forge-cli` and `forge-cli --local-swarm`, can we have a subcommand that explicitly names the backend? e.g. `forge-cli k8s <args>` and `forge-cli local <args>`

Reply:
yea, we can do all those cli improvement by this time, I will close this PR and create a new one